src/injector.ts

```diff
@@ -188,18 +189,23 @@ export default class Injector {
    * and by swarm | ipfs hash.
    * @param  {string}            repository repository root (ex: 'repository')
    * @param  {string}            chain      chain name (ex: 'ropsten')
-   * @param  {string}            address    contract address
+   * @param  {string}            address    contract address or ENS address
+   * @param  {boolean}           isENS      needs to be resolved by ENS
    * @param  {string[]}          files
    * @return {Promise<string[]>}            addresses of successfully verified contracts
    */
   public async inject(
     repository: string,
     chain: string,
     address: string,
+    isENS: boolean,
     files: string[]
   ) : Promise<string[]> {
 
     if (address) {
+      if (isENS) {
```

Comment:
Another way this could be achieved is keeping the assumption the input is not sanitized and match domains here, not sure if we should be accepting subdomains...